src/actions/classifier.js

```diff
@@ -72,6 +72,13 @@ export function saveAnnotation(task, value) {
   }
 }
 
+export function removeAnnotationValue(task, value) {
+  return (dispatch, getState) => {
+    const workflowID = getState().classifier.currentWorkflowID
+    dispatch(removeState(`classifier.annotations.${workflowID}.${task}`, value))
```

Comment:
Does this remove the original annotation value (yes or no) and replace it with the shortcut value? The shortcut component was recently updated to retain the original annotation/answer and add that unlinked task as an additional answer.